Libplanet/Blockchain/TotalSupplyStateCompleters.cs

```diff
@@ -51,7 +51,11 @@ public static class TotalSupplyStateCompleters<T>
         /// </summary>
         public static readonly TotalSupplyStateCompleter<T> Reject =
             (chain, blockHash, currency) =>
-                throw new IncompleteBlockStatesException(blockHash);
```

Comment:
I think that discarding other parameters (i.e., `chain`, `currency`) may be enough. (of course, we may need to suppress stylecop with `#pragma`... 😢 )

Reply:
I personally dislike using `#pragma` since the code clip remains in the business logic. I don't expect the case but if there are any chances of intrusive ways to avoid the problem.